frontend/src/pages/meditation/components/meditation-modal/meditation-modal.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Button,
+  Input,
+  InputFile,
+  Modal,
+} from '#libs/components/components.js';
+import { EMPTY_ARRAY_LENGTH } from '#libs/constants/constants.js';
+import { DataStatus } from '#libs/enums/enums.js';
+import {
+  useAppForm,
+  useAppSelector,
+  useCallback,
+  useEffect,
+  useState,
+} from '#libs/hooks/hooks.js';
+import {
+  createMeditationValidationSchema,
+  type MeditationCreateValidation,
+} from '#packages/meditation/meditation.js';
+import { DEFAULT_MEDITATION_PAYLOAD } from '#pages/meditation/libs/constants/constants.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  isDisplayed: boolean;
+  setIsDisplayed: React.Dispatch<React.SetStateAction<boolean>>;
+  onSubmit: (title: string, file: File) => void;
+  onClose: () => void;
+};
+
+const MeditationModal: React.FC<Properties> = ({
+  isDisplayed,
+  setIsDisplayed,
+  onSubmit,
+  onClose,
+}) => {
+  const [file, setFile] = useState<File | null>(null);
```

Comment:
@Liza-Veis I tried pass file directly the same way and got joi error, I mentioned about it in [internal chat](https://bsa2023.slack.com/archives/C05P563E2MN/p1694020754883559) and the problem still not resolved. Otherwise I need to send parameters partially and save whole file in a state as it's done right now.

Reply:
@v0ldemar01 Okay, I'll try. At now I stuck in searching joi files, it may take some time to find the solution